cnf-certification-test/webserver/index.html

```diff
@@ -270,43 +275,68 @@ <h1>CNF Certification Test</h1>
         
         </fieldset>
       </div>
-
         <fieldset>
           <legend>Select a Test</legend>
-          <label>
+
+          <legend id="selectComboBox"> Selcet a Test Tag <select name="class" id="selectScenarioComboBox" onchange="selectScenario()">
```

Comment:
initialize the scenarios when loading the page for the first time, not only on change, maybe with all tests selected